public/assets/css/blogpost.css

```diff
@@ -0,0 +1,206 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 95%;
+  min-height: 100vh;
+}
+header {
+  width: 100%;
+}
+
+.card-wrap {
+  width: 100%;
+}
+
+.blog-card {
+  padding: 60px;
+  max-width: 1100px;
+  margin: auto;
+  margin-top: 50px;
+  color: var(--text-secondary);
+}
+
+.blog-card h1,
+.blog-card h2,
+.blog-card h3,
+.blog-card h4,
+.blog-card h5,
+.blog-card h6,
+.blog-card strong,
+.blog-card a,
+.blog-card a * {
```

Comment:
Why do you need so many selectors? Aside from the anchor, wouldn't just `.blog-card` be sufficient?